blocks/address/src/app.tsx

```diff
@@ -410,7 +431,7 @@ export const App: BlockComponent<RootEntity> = ({
                 <Box sx={{ display: "flex", columnGap: 3, flexWrap: "wrap" }}>
                   <Link
                     //  @todo: link this to the block's hub page
-                    href=""
+                    href="https://blockprotocol.org/@hash/blocks/address"
```

Comment:
We can remove the `@todo` now, right?

Reply:
Thanks for catching 0f6baee8618547a9897638bde3c07ec425be07ba